### PR TITLE
Fix crashes when reopening app

### PR DESCRIPTION
**Pull request by xMasterX**
Fix crash when closing and opening app second time
(Proper deinit of lpuart bus)

### CODE REVIEW

**Review by 0xchocolate — APPROVED** [review]

**Review by Mjefoester — COMMENTED** [review]
😀

**Review by Mjefoester — COMMENTED** [review]
thank you

File: wifi_marauder_uart.c [hunk 1]
```diff
@@ -106,7 +106,10 @@ void wifi_marauder_uart_free(WifiMarauderUart* uart) {
...
     furi_hal_uart_set_irq_cb(uart->channel, NULL, NULL);
+     if(uart->channel == FuriHalUartIdLPUART1) {
+        furi_hal_uart_deinit(uart->channel);
+    }
     furi_hal_console_enable();
 
     free(uart);
-}
\ No newline at end of file
+}
```
Comment (Mjefoester) [inline wifi_marauder_uart.c]: 😄

**Review by Mjefoester — COMMENTED** [review]
thanks

**Review by Mjefoester — COMMENTED** [review]
thank you

**Pull request merged**